src/wasm2c.c.tmpl

```diff
@@ -22,8 +26,12 @@
        ? ((t)table.data[x].func)(__VA_ARGS__)        \
        : TRAP(CALL_INDIRECT))
 
+#ifndef WASM_RT_NO_MEMORY_CHECKS
```

Comment:
maybe `WASM_RT_NO_MEMORY_BOUNDS_CHECKS` instead? Not sure what other memory checks we may have, but seems better to be specific.

Reply:
sounds good.